doc/source/whatsnew/v1.0.0.rst

```diff
@@ -297,8 +297,9 @@ Missing
 MultiIndex
 ^^^^^^^^^^
 
+
 - Constructior for :class:`MultiIndex` verifies that the given ``sortorder`` is compatible with the actual ``lexsort_depth``  if ``verify_integrity`` parameter is ``True`` (the default) (:issue:`28735`)
--
+- If `MultiIndex` nlevel is 3 and first level index in `MultiIndex` is `PeriodIndex` , `.loc` could not raise exception when key is not matched (:issue:`20684`)
```

Comment:
this note is too specific, just say using .loc with a PeriodIndex as a level in a MultiIndex